cmd/statusd/main.go

```diff
@@ -332,7 +332,7 @@ Options:
 // haltOnInterruptSignal catches interrupt signal (SIGINT) and
 // stops the node. It times out after 5 seconds
 // if the node can not be stopped.
-func haltOnInterruptSignal(nodeManager common.NodeManager) <-chan struct{} {
+func haltOnInterruptSignal(nodeManager *node.NodeManager) <-chan struct{} {
```

Comment:
`node.Manager` seems like a better name here

Reply:
Hey agree, thats for sure something i'll be addressing on [Iteration 6](https://github.com/status-im/ideas/issues/63).  Just trying to have small prs :)

Reply:
Agree, but this change seems to be trivial. You can put it in a separate commit if you want.

Reply:
Renamed `node.NodeManager` to `node.Manager`